cpp/csp/python/PyPushInputAdapter.cpp

```diff
@@ -202,7 +202,8 @@ struct PyPushInputAdapter_PyObject
 };
 
 static PyMethodDef PyPushInputAdapter_PyObject_methods[] = {
-    { "push_tick", (PyCFunction) PyPushInputAdapter_PyObject::pushTick, METH_VARARGS, "push new tick" },
+    { "push_tick",          (PyCFunction) PyPushInputAdapter_PyObject::pushTick, METH_VARARGS, "push new tick" },
+    { "_engine_shutdown",   (PyCFunction) PyEngine_shutdown<PyPushInputAdapter_PyObject>,  METH_VARARGS, "engine shutdown" },
```

Comment:
I dont see any reason to make it a "hidden" method
I think shutdown_engine() would be a better name

Reply:
Done, renamed